### PR TITLE
Add utility to connect to a Chrome instance.

### PR DESCRIPTION
**Pull request by diasbruno**
Hi,

Add utility to connect to a chrome instance.
 
- `indium-chrome-connect-to-url`

Related issue:
- #134

### CODE REVIEW

**Comment by diasbruno** [comment]
This one should be ready for review.

File: indium-chrome.el [hunk 1]
```diff
@@ -113,6 +113,21 @@ Try a maximum of NUM-TRIES."
           (port (read-from-minibuffer "Port: " (number-to-string indium-chrome-port))))
       (indium-chrome--get-tabs-data host port #'indium-chrome--connect-to-tab))))
 
+;;;###autoload
+(defun indium-connect-to-local-chrome-to-url (url)
```
Comment (NicolasPetton) [inline indium-chrome.el]: What about making the `HOST` and `PORT` parameters optional in `indium-chrome-connect-to-chrome-url` instead, defaulting to localhost:9229?

File: indium-chrome.el [hunk 2]
```diff
@@ -113,6 +113,21 @@ Try a maximum of NUM-TRIES."
...
+  "Connect to a local Chrome instance to the given tab URL."
+  (indium-chrome--connect-to-chrome-to-url "localhost" "9229" url))
+
+;;;###autoload
+(defun indium-connect-to-chrome-to-url (host port url)
```
Comment (NicolasPetton) [inline indium-chrome.el]: What about naming it `indium-chrome-connect-to-url`?

File: indium-chrome.el [hunk 3]
```diff
@@ -113,6 +113,21 @@ Try a maximum of NUM-TRIES."
...
+  "Connect to a Chrome instance with HOST:PORT to a given tab URL."
+  (indium-chrome--connect-to-chrome-to-url host port url))
+
+(defun indium-chrome--connect-to-chrome-to-url (host port url)
```
Comment (NicolasPetton) [inline indium-chrome.el]: Could this function be inlined?

Comment (diasbruno) [reply indium-chrome.el]: I'll leave for now to make the tests pass. Let me know if there is a better way.

File: indium-chrome.el [hunk 4]
```diff
@@ -121,6 +130,13 @@ Try a maximum of NUM-TRIES."
                                         nil
                                       (indium-chrome--read-tab-data))))))
 
+(defun indium-chrome--connect-to-url (host port url)
```
Comment (NicolasPetton) [inline indium-chrome.el]: is this one still used?

Comment (diasbruno) [reply indium-chrome.el]: Yeah, I let this one to make it ease to test. Let me know if there is a better way.

Comment (diasbruno) [reply indium-chrome.el]: Later, today, I'll give it a try to get this correctly.

**Comment by sandric** [comment]
Guys, I know thats not how you suppose to interact in open source world, but please, make this pr merged, and functionality available. Just imagine what doors it will open with simple automatization you will be able to click shortcut in chrome tab, and it will open emacs buffer with console in it for that pane.

**Comment by ebpa** [comment]
Whoops!  I created #157 before I saw this PR.

IMHO, an additional function (`indium-chrome-connect-to-url`/`indium-chrome--connect-to-url`) is unnecessary; the library already has `indium-connect-to-chrome` which does just that (it connects to the chrome instance using a hostname and port).  I vote that those parameters be exposed on the function signature and to use an interactive form.

See:
https://github.com/NicolasPetton/Indium/pull/157/commits/8bc8a882fc8dfe5e28f4062e56569fddb4a7f2fd#diff-cddac571bae142867141ce263f5d9eddR105

**Pull request closed without merging**